.pre-commit-config.yaml

```diff
@@ -0,0 +1,49 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.1.0
+    hooks:
+    -   id: check-yaml
+    -   id: debug-statements
+    -   id: name-tests-test
+    -   id: requirements-txt-fixer
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-docstring-first
+    -   id: double-quote-string-fixer
+    -   id: check-ast
+    -   id: no-commit-to-branch
+    -   id: check-merge-conflict
+
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.2
+    hooks:
+    -   id: flake8
+
+-   repo: https://github.com/asottile/add-trailing-comma
+    rev: v1.2.0
+    hooks:
+    -   id: add-trailing-comma
+
+-   repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.1.6
+    hooks:
+    -   id: forbid-crlf
+    -   id: remove-crlf
+    -   id: forbid-tabs
+    -   id: remove-tabs
+        args: [ --whitespaces-count, '4' ]  # defaults to: 4
+
+-   repo: git://github.com/chewse/pre-commit-mirrors-pydocstyle
+    rev: 22d3ccf
+    hooks:
+    -   id: pydocstyle
+
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.701
+    hooks:
+    -   id: mypy
+
+-   repo: https://github.com/pre-commit/mirrors-pylint
+    rev: v2.3.1
+    hooks:
+    - id: pylint
```

Comment:
The task was to use a "local" mode for this linter. Here's an example: https://github.com/sanitizers/octomachinery/blob/93899d4/.pre-commit-config.yaml#L51-L58